utils/scripts/find_duplicate_read_sets/find_duplicate_read_sets.py

```diff
@@ -0,0 +1,118 @@
+                        } 
+                        df = pd.concat([df, pd.DataFrame([s1])])
+                    if etag_source_2:
+                        s2 = {
+                            'sequence_store_id': seq_store,
+                            'read_set_id': read_set_id, 
+                            'name': name,
+                            'type': file_type,
+                            'source': 'source 2', 
+                            'etag': etag_source_2
+                        } 
+                        df = pd.concat([df, pd.DataFrame([s2])])
+
+    # filters to only duplicates based on the etag field
+    filtered_df = df[df.duplicated(subset='etag', keep=False)]
+
+    # sorts for easy identification
+    df_sorted = filtered_df.sort_values(by=['etag','read_set_id'], ascending=True)
+
+    return df_sorted
+
+
+
+# Driver Code
+
+# Decide the two file paths according to your
+# computer system
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('-s', '--seq_store_ids', type=str, required=True, dest='seqStoreIds', 
+                        help='A comma separated list of 1 or more sequence store IDs to check, all in the same region.')
+    parser.add_argument('-r', '--region', type=str, required=True, dest='region', 
+                        help='The region where the sequence stores are in. They must all be in the same region.')
+    parser.add_argument('-o', '--output', type=str, default='duplicate_read_sets.csv', dest='output', 
+                        help='(optional) The path and file name for the output csv file containing all the duplicate read sets. Include the file \
+                        name and prefix (e.g. "~/path/to/out/duplicates.csv"). The default file name is duplicate_read_sets.csv and will write to \
+                        the current directory.')
+    parser.add_argument('-p', '--profile-name', type=str, default='NA', dest='profileName', 
+                        help='(optional) The profile name for boto3 to use if you do not want it to use the default profile configured.')
+    
+    args = parser.parse_args()
+
+    # parse the input sequence store IDs so it's clear what will get passed to the function. 
+    parsed_seq_store_ids = [i.strip() for i in args.seqStoreIds.split(',')]
+    if len(parsed_seq_store_ids) == 0:
+        sys.exit(f'\nERROR: could not parse sequence store ids input: {args.seqStoreIds}')
+
+    print('\nArguments used:\nSequence Store IDs:', str(parsed_seq_store_ids))
+    print('Region:', args.region)
+    print('Output:', args.output)
+
+    # print optional inputs only if they're specified
+    if args.profileName and args.profileName != 'NA':
+        print('Profile Name:', args.profileName,'\n')
+
+    # setup the boto3 client
+    if args.profileName and args.profileName != 'NA':
+        boto3.setup_default_session(profile_name=args.profileName)
+
+    omics_client = boto3.client('omics', region_name=args.region)
```

Comment:
Recommend using a `boto3.Session` object to create the client. This simplifies calling with a user specified region and profile. For example:

```python
session = boto3.Session(region_name=args.region, profile_name=args.profile)
omics = session.client('omics')
```

Reply:
done